FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Services/AuthService.swift

```diff
@@ -119,7 +131,7 @@ public final class AuthService {
     get throws {
       guard let provider = facebookProvider else {
         throw AuthServiceError
-          .notConfiguredProvider("`FacebookProviderSwift` has not been configured")
+          .notConfiguredProvider("`FacebookProviderAuthUI` has not been configured")
```

Comment:
Should this not just be a `fatalError` given consumers won't have conditional configuration of providers?

Reply:
Yeah, that makes sense. Updated: [`6b08073` (#1256)](https://github.com/firebase/FirebaseUI-iOS/pull/1256/commits/6b080730ec42991b88aefec55a1e71e1b4627172)